src/main/java/Menu.java

```diff
@@ -0,0 +1,151 @@
+public class Menu {
+    private Task[] tasks;
+    private int taskCount;
+    private static final String DIVIDER = "____________________________________________________________";
+    private static final int MAX_TASK_SIZE = 100;
```

Comment:
Pro tip - You can implement an array list instead of a fixed size array so that the application can be re-purposed to help the user complete more tasks.